packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -73,45 +75,94 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
          * @returns {Promise<Invitation<OperatorKit>>}
          */
         makeOperatorInvitation(operatorId) {
-          const { admin } = this.facets;
+          const { creator } = this.facets;
           trace('makeOperatorInvitation', operatorId);
 
           return zcf.makeInvitation(
             /** @type {OfferHandler<OperatorKit>} */
             seat => {
               seat.exit();
-              return admin.initOperator(operatorId);
+              return creator.initOperator(operatorId);
             },
             INVITATION_MAKERS_DESC,
           );
         },
+        /** @param {string} operatorId */
+        async initOperator(operatorId) {
+          const { operators, pending } = this.state;
+          trace('initOperator', operatorId);
+
+          const operatorKit = makeOperatorKit(
+            operatorId,
+            this.facets.operatorPowers,
+          );
+          operators.init(operatorId, operatorKit);
+          pending.init(operatorId, makeScalarBigMapStore('pending evidence'));
+
+          return operatorKit;
+        },
+
         /** @param {string} operatorId */
         async removeOperator(operatorId) {
-          const { operators: oracles } = this.state;
+          const { operators } = this.state;
           trace('removeOperator', operatorId);
-          const kit = oracles.get(operatorId);
-          kit.admin.disable();
-          oracles.delete(operatorId);
+          const operatorKit = operators.get(operatorId);
+          operatorKit.admin.disable();
+          operators.delete(operatorId);
         },
       },
+      operatorPowers: {
+        /**
+         * Add evidence from an operator.
+         *
+         * @param {CctpTxEvidence } evidence
+         * @param {OperatorKit} operatorKit
+         */
+        submitEvidence(evidence, operatorKit) {
+          const { pending } = this.state;
+          trace(
+            'submitEvidence',
+            operatorKit.operator.getStatus().operatorId,
+            evidence,
+          );
+          const { operatorId } = operatorKit.operator.getStatus();
 
-      admin: {
-        /** @param {string} operatorId */
-        async initOperator(operatorId) {
-          const { operators: oracles } = this.state;
-          trace('initOperator', operatorId);
+          // TODO verify that the operator is one made by this exo
+          // This doesn't work...
+          // operatorKit === operators.get(operatorId) ||
```

Comment:
That rings a bell... exoKits don't have stable identity. Their facets do, though:

```suggestion
          operatorKit?.facets?.admin === operators.get(operatorId).facets.admin ||
```

Reply:
Right, but `.facets` is internal to the exo. When I passed `this.facets` as the argument to this function, that worked in unit tests but failed in the contract test.